### PR TITLE
Relax 'engines' dependency to allow for any version of node past 6.11.2

### LINKED ISSUE #19: Package hardcoded to only work on Node <7
Hi Yves.  We'd like to use this great package.  However, we're running into a problem due to it having a hard blocker on the version of hte node engine being <7.  Our system works by allowing one to compile with a local version of node (that can effectively be pretty much any version), which then will finally run on Azure (which also supports 8.11.1 and 10.6.0).  

This blocker prevents us from supporting that.  Is there a reason this needs to be in here?  From looking at your code, i can't see anything in it that is particularly node version specific. 

Would you be willing to remove this upper-bound blocker?  I'd be happy to submit the PR for it if so.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Closes #19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "validate": "npm run style && npm run coverage"
   },
   "engines": {
-    "node": ">=6.11.2 <7",
+    "node": ">=6.11.2",
```
Comment (yvele) [inline package.json]: Actually we support node `<11` see https://github.com/yvele/azure-function-express/pull/18#issuecomment-417846637

I think it's better to make it clear that the code has actually been tested for node `>=6.11.2 <11`

When the future node 12 version come out, we will do another PR with extended tests 😉

So at the moment I prefer `>=6.11.2 <11` over `>=6.11.2` to be more cautious and explicit.

Comment (CyrusNajmabadi) [reply package.json]: That's fine with me :)

**Comment by CyrusNajmabadi** [comment]
Anything else you'd like me to do here?  Thanks!

**Comment by yvele** [comment]
Nope I think that's good 👍

**Pull request merged**